Python 2 syntax error in merge_overlapping_intervals.py

flake8 testing of https://github.com/ealter/vim_turing_machine on Python 2.7.13

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./vim_turing_machine/machines/merge_overlapping_intervals/merge_overlapping_intervals.py:343:17: E999 SyntaxError: invalid syntax
                *self.move_to_blank_spaces(
                ^
```

--- a/README.md
+++ b/README.md
@@ -100,6 +100,14 @@ it not to auto step and then step through manually using `y$@"`.
 
 Happy hacking!
 
+Dependencies
+============
+
+To run this code, you will need `python3.6`, `tox`, and `vim` installed on your
+machine. This code hasn't been tested on other versions of python3, but they'll
+probably work if you change the pinned version in `tox.ini`. This code is not
+python2 compatible.
+
 Contributors
 ============
 

--- a/setup.py
+++ b/setup.py
@@ -6,8 +6,6 @@ setup(
     name='vim-turing-machine',
     version='1.0.0',
     classifiers=[
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.6',
     ],

